enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/validator/RevokedJWTDataHolder.java

```diff
@@ -36,10 +35,8 @@
     private static RevokedJWTDataHolder instance = new RevokedJWTDataHolder();
 
     public void init() {
-        if (ConfigHolder.getInstance().getConfig().getEventHub().isEnabled()) {
-            RevokedTokenDiscoveryClient revokedTokenDs =  RevokedTokenDiscoveryClient.getInstance();
-            revokedTokenDs.watchRevokedTokens();
-        }
+        RevokedTokenDiscoveryClient revokedTokenDs = RevokedTokenDiscoveryClient.getInstance();
```

Comment:
this means we connect this xds whether controlplane is enabled or not. is that the expected behavior?

Reply:
I think its wrong to check the control plane enabled config at the Enforcer. Connecting the service should be fine since we share the same host and port right?